src/integration/java/io/pinecone/integration/controlPlane/pod/CollectionTest.java

```diff
@@ -77,7 +73,6 @@ public void testIndexFromCollectionHappyPath() throws InterruptedException {
 
         assertEquals(collection.getStatus(), CollectionModel.StatusEnum.READY);
         assertEquals(collection.getDimension(), dimension);
-        assertEquals(collection.getVectorCount(), 3);
```

Comment:
Since we're now using a shared index across tests and there's no guarantee around how many vectors may be in the index by the time this test runs, I removed this check for now. 

We validate the actual vectors in the collection using `upsertIds` below which feels like it should be sufficient for validating the contents.